src/tree/LocalRootTreeItemBase.ts

```diff
@@ -342,7 +346,12 @@ export abstract class LocalRootTreeItemBase<TItem extends ILocalItem, TProperty
         let isDockerStatusChanged = false;
 
         try {
-            this._itemsFromPolling = await this.getSortedItems();
+            const pollingItems = await this.getSortedItems();
```

Comment:
If refresh is suspended (Docker context switch in progress), you do not need to make this call.

Reply:
Before `hasChanged()` is called we do check if refresh is suspended. But this is to handle the case where the refresh is suspended while waiting on the result of `this.getSortedItems()`.

Reply:
Should we then check both before and after the call?